drupal/sites/default/boinc/themes/boinc/templates/views-view-table--boinc-account-computers--panel-pane-1.tpl.php

```diff
@@ -35,6 +35,6 @@
     <?php print l(bts('More', array(), NULL, 'boinc:account-dashboard'), 'account/computers'); ?>
   </li>
   <li class="first alt tab">
-    <?php print l(bts('Tasks', array(), NULL, 'boinc:host-tasks-link'), 'account/tasks/active'); ?>
+    <?php print l(bts('Tasks', array(), NULL, 'boinc:host-tasks-link'), 'account/tasks/0/1'); ?>
```

Comment:
Yes. In fact changing that link is now part of DBOINCP-71. I will generate a new PR today.